Sources/PreviewGallery/PreviewGallery.swift

```diff
@@ -26,25 +26,37 @@ struct PreviewCellView: View {
   }
 }
 
+/// A SwiftUI View that displays a gallery of previews organized by modules.
+///
+/// `PreviewGallery` presents a list of modules, each containing its respective previews.
+/// If no previews are found, it displays a message indicating so.
 public struct PreviewGallery: View {
-  
-  let data: PreviewData
-  
-  @MainActor
-  public init(data: PreviewData? = nil) {
-    self.data = data ?? .default
-  }
-  
-  public var body: some View {
-    if data.modules.count > 0 {
-      List {
-        ForEach(Array(data.modules).sorted(), id: \.self) { module in
-          ModulePreviews(module: module, data: data)
+    /// The data source containing preview information.
+    let data: PreviewData
+
+    /// Initializes a new `PreviewGallery` with the given preview data.
+    ///
+    /// This initializer must be called on the main actor because it potentially
```

Comment:
This seems like a very specific implementation detail that shouldn't be in the public docs. It's a SwiftUI `View` so it's natural to init it on the MainActor anyway